v2/cmd/samples/httpb/requester/main.go

```diff
@@ -29,14 +30,17 @@ func main() {
 			"message": "Hello, World!",
 		})
 
-		resp, err := c.Request(ctx, e)
-		if err != nil {
-			log.Printf("failed to send: %v", err)
+		resp, result := c.Request(ctx, e)
```

Comment:
Most of the samples that use client and assume Send/Request throw errors for non nil results need to be updated.

Reply:
added an issue.